go/libcmdline/cmdline.go

```diff
@@ -352,6 +367,18 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Name:  "api-timeout",
 			Usage: "set the HTTP timeout for API calls to the keybase API server",
 		},
+		cli.StringFlag{
+			Name:  "tor-mode",
+			Usage: "set TOR mode to be 'leaky', 'none', or 'strict'. 'none' by default",
```

Comment:
I'm fine with anything, as long as there's some way for the user reading this help string to figure out how to get to the definition of these modes.